### DISCUSSION TITLE
second screen - vhs effect

### DISCUSSION THREAD

**Discussion by jjonojj** [i3/i3#6023] [Q&A]
Hi.
after using hyprland for a long time, i decided to switch to i3.
i did a fresh install of endeavour os with i3, and my second monitor looked like it had an vhs effect on it. it was flickering and on the left side there was a vertical green line going down the screens left edge. after that i tried bare arch (install over archinstall command), but the issue was still appearing. any help is appreciated.

**Comment by orestisfl** [comment]
Sounds like a driver problem or a faulty monitor. What hardware are you
using and which drivers?

**Reply by jjonojj** [reply]
i just installed xfce for testing and the issue is still present. i will close this and continue it in the r/linuxquestions subreddit.
